src/Sulu/Bundle/AdminBundle/DependencyInjection/SuluAdminExtension.php

```diff
@@ -135,6 +135,9 @@ public function prepend(ContainerBuilder $container)
                         ],
                     ],
                 ],
+                'icon_sets' => [
+                    'sulu' => 'icomoon://' . __DIR__ . '/../Resources/js/components/Icon/selection.json',
```

Comment:
I think it would make sense to also prepend the config for a svg set in the public directory, e.g. 
`'website => 'svg://%kernel.project_dir%/public/icons/'` 

WDYT @sulu/core-developer

Reply:
No it need be a valid path we should not configure something not existing.

Reply:
Yes, but we could create an empty directory, I just thought introducing a configuration might make it more easy to use, especially since, as far as I know, this would be the first content type that requires a configuration

but only using the sulu icons is also fine for me 😄

Reply:
would go only with the sulu icons, as accessing the filesystem if not required should be avoided. Think that most projects may not require the icon type.